src/lib/components/chat/Settings/Models.svelte

```diff
@@ -466,6 +467,17 @@
 		ollamaVersion = await getOllamaVersion(localStorage.token).catch((error) => false);
 		liteLLMModelInfo = await getLiteLLMModelInfo(localStorage.token);
 	});
+	const deleteModelPull = async(model: string) => {
+		const {reader} = modelDownloadStatus[model];
+		if(reader){
+			await reader.cancel();
+			delete modelDownloadStatus[model];
+			await deleteModel(localStorage.token, model);
```

Comment:
Calling delete endpoint with `mistral` fails with 

```json
{
    "detail": "Model 'mistral' was not found"
}
```

What if a tag is not provided, should I assume `:latest` and just append that to the `model` ? 

For example: 
- if the input is `mistral` then add `:latest` => `mistral:latest` 
- if the input is `mistral:7b` then just send the data

Reply:
I'm not sure if Ollama provides APIs for deleting partially downloaded blobs, we might want to look into that!

Reply:
I'll merge this for now, but let's come back to this later. Thanks!